packages/ui/src/routes/settings/AddTokensPage.tsx

```diff
@@ -41,7 +41,7 @@ const AddTokensPage = () => {
 
     const isManualTokenView = utils.isAddress(searchedValue)
     useEffect(() => {
-        if (searchedValue) {
+        if (searchedValue && searchedValue !== "" && !isManualTokenView) {
```

Comment:
Why do we have to add the `searchedValue !== ""`? Isn't enough checking the `searchedValue`?

Reply:
💯